libs/SalesforceSDKCore/SalesforceSDKCore/Classes/Security/SFAuthenticationManager.h

```diff
@@ -470,18 +470,30 @@ extern  NSString * const kOAuthRedirectUriKey;
 + (BOOL)errorIsInvalidAuthCredentials:(NSError *)error;
 
 /**
- Remove any cookies with the given names from the given domains.
- @param cookieNames The names of the cookies to remove.
+ Remove all cookies for the given domains. Fire and Forget.
  @param domainNames The names of the domains where the cookies are set.
  */
-+ (void)removeCookies:(NSArray<NSString*> *)cookieNames fromDomains:(NSArray<NSString*> *)domainNames;
++ (void)removeCookiesFromDomains:(NSArray<NSString*> *)domainNames;
```

Comment:
Do we need this API? We could simply have the API below with the completion block being optional (`nullable`).